### PR TITLE
bob: add page

### PR DESCRIPTION
**Pull request by Kristmh**
Add new page for bob with some simple commands.

- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages/common/bob.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+`bob use stable`
+
+- Install and switch to nightly version of Neovim:
+
+`bob use nightly`
+
+- List installed and currently used versions of Neovim:
+
+`bob ls`
```
Comment (acuteenvy) [inline pages/common/bob.md:16]:
```suggestion
`bob list`
```

File: pages/common/bob.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+- Uninstall Neovim and erase any changes bob has made:
```
Comment (acuteenvy) [inline pages/common/bob.md:18]:
```suggestion
- Uninstall Neovim and erase any changes `bob` has made:
```

File: pages/common/bob.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+`bob erase`
+
+- Rollback nightly version:
```
Comment (acuteenvy) [inline pages/common/bob.md:22]:
```suggestion
- Roll back to a previous nightly version:
```

File: pages/common/bob.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+> Bob is a Neovim version manager that makes it easy to switch between Neovim versions.
+> More information: <https://github.com/MordechaiHadad/bob>.
+
+- Install and switch to stable version of Neovim:
+
+`bob use stable`
+
+- Install and switch to nightly version of Neovim:
+
+`bob use nightly`
```
Comment (acuteenvy) [inline pages/common/bob.md:12]:
```suggestion
- Install and switch to the specified version of Neovim:

`bob use {{nightly|stable|latest|version_string|commit_hash}}`
```

File: pages/common/bob.md [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+
+- List installed and currently used versions of Neovim:
+
+`bob ls`
+
```
Comment (acuteenvy) [inline pages/common/bob.md:17]:
```suggestion

- Uninstall the specified version of Neovim:

`bob uninstall {{nightly|stable|latest|version_string|commit_hash}}`

```

**Review by acuteenvy — COMMENTED** [review]
Thanks for your contribution. I have some suggestions, but you should probably wait for the maintainers to approve them.

**Review by kbdharun — COMMENTED** [review]
Thanks for your contribution, LGTM after @acuteenvy's suggestions are addressed (Great review btw).

**Comment by Kristmh** [comment]
> Thanks for your contribution, LGTM after @acuteenvy's suggestions are addressed (Great review btw).

Thank you for the review @acuteenvy and @kbdharun . Keep up the good work :)

**Review by einverne — APPROVED** [review]
Looks good to me after @acuteenvy 's suggestion.

**Review by kbdharun — APPROVED** [review]

**Pull request merged**